frontend/src/components/PricingPage.jsx

```diff
@@ -1,29 +1,185 @@
+import { useState } from 'react';
+
 export const PricingPage = () => {
+  const businessPriceFloor = 10;
+  const businessPricePerHead = 20;
```

Comment:
I actually think it's kinda cool the price info is in the code :)

Reply:
Agree!